partiql-parser/src/main/antlr/PartiQLParser.g4

```diff
@@ -201,6 +201,26 @@ querySpecification
         where=whereClauseSelect?
         group=groupClause?
         having=havingClause?
+        window=windowClause?
+    ;
+
+/**
+ * EBNF 2023:
+ * <window clause> ::= WINDOW <window definition list>
+ * <window definition list> ::= <window definition> [ { <comma> <window definition> }... ]
+ */
+windowClause
+    : WINDOW windowDefinition (COMMA windowDefinition)*
+    ;
```

Comment:
Just a personal notes to remind myself when review further section: 

It seems like we can parse statement like 
```
SELECT w1 FROM employee AS t WINDOW w1 AS (PARTITION BY t.b ORDER BY t.c);
```

Curious if we support this.